content/en/logs/indexes/_index.md

```diff
@@ -24,83 +24,102 @@ Indexes are located on the [Configuration page][1] in the Indexes section. Doubl
+By default, Log Explorer have one unique Log Index, but datadog also offers multiple indexes if you require:
 
-**Note**: If a log matches several exclusion filters, only the first exclusion filter rule is applied. A log is not sampled or excluded multiple times by different exclusion filters.
+* Multiple retention periods and/or multiple [daily quotas](#set-daily-quota), for finer budget control.
+* Multiple permissions, for finer user [role based access controls (RBAC)][7].
 
-To define a new index filter, click on the "add" button:
+<div class="alert alert-info">
+The multi-index feature is in private beta, <a href="/help">contact Datadog support</a> to turn it on for your account.
+</div>
 
-{{< img src="logs/indexes/index_filters.png" alt="index filters"  style="width:70%;">}}
+## Indexes Filters
 
-To configure an exclusion filter:
+Index filters allow dynamic control over which logs flow into which indexes.  For example, if you create a first index filtered on the `status:notice` attribute, a second index filtered to the `status:error` attribute, and a final one without any filter (the equivalent of `*`), all your `status:notice` logs would go to the first index, all your `status:error` logs to the second index, and the rest would go to the final one.
 
-1. Define the name of your filter.
-2. Define the query for logs to exclude from your index.
-    **Note**: It is possible to use any attribute or tag in the index filter query, even those that are not facets. If you are filtering by non-faceted attributes or tags, be sure to hit "enter/return" from the query bar.
-3. Define the sampling rate.
-4. Save the filter.
+{{< img src="logs/indexes/multi_index.png" alt="Multi indexes"  style="width:70%;">}}
 
-    {{< img src="logs/indexes/index_filter_details.png" alt="index filter details"  style="width:80%;">}}
+**Note**: **Logs enter the first index whose filter they match on**, use drag and drop on the list of indexes to reorder them according to your use-case.
 
-### Reorder filters
+## Exclusion Filters
 
-Order matters for exclusion filters. Contrary to how several pipelines can process a log, if a log matches several exclusion filters, only the first exclusion filter rule is applied.
+By default, logs indexes have no exclusion filter: meaning all logs matching the [Index Filter](#indexes-filters) are indexed.
 
-Reorder your pipeline to make sure the proper exclusion filters apply to your log. For instance, you probably want to set up the filters ordered by least inclusive to most inclusive queries..
+But because your logs are not all and equally valuable; exclusion filters control which logs flowing in your index should be removed. Excluded logs are discarded from indexes, but still flow through the [Livetail][8] and can be used to [generate metrics][9] and [archived][10].
```

Comment:
```suggestion
But because your logs are not all and equally valuable, exclusion filters control which logs flowing in your index should be removed. Excluded logs are discarded from indexes, but still flow through the [Livetail][8] and can be used to [generate metrics][9] and [archived][10].
```